### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
## Mobile version skeleton update

in this update we:

- Added a works section
- Added an about section
- Added a skills section
- Added a contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+                <div class="about">
+                    <h1>About me</h1>
+                    <p class="description">
+                        Hello I'm a software developer! I can help
+                        you build a product , feature or website 
+                        Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded,
+                        don't hestiate to contact me.
+                    </p>
+                    <button type="submit">Get My Resume</button>
```
Comment (mwanawabangona) [inline index.html:210]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 2]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+                                </ul>
+                                <ul class="block2-tag2">
+                                    <li>bootstrap</li>
+                                </ul>
+                                <ul class="block2-tag3">
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="button-container"><button type="submit" class="action">See Project</button></div>
+                        </div>
```
Comment (mwanawabangona) [inline index.html:197]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 3]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+                                    <li>html</li>
+                                </ul>
+                                <ul class="block2-tag2">
+                                    <li>bootstrap</li>
+                                </ul>
+                                <ul class="block2-tag3">
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="button-container"><button type="submit" class="action">See Project</button></div>
```
Comment (mwanawabangona) [inline index.html:174]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 4]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+                        </div>
```
Comment (mwanawabangona) [inline index.html:153]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 5]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
```
Comment (mwanawabangona) [inline index.html:131]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 6]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
```
Comment (mwanawabangona) [inline index.html:109]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 7]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
```
Comment (mwanawabangona) [inline index.html:87]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 8]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+                            </ul>
+                            <ul class="tag3">
+                                <li>bootstrap</li>
+                            </ul>
+                            <ul class="tag4">
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button type="submit" class="button">See Project</button>
+                    </div>
```
Comment (mwanawabangona) [inline index.html:66]: - [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one

File: index.html [hunk 9]
```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+            <div class="contact-form">
+                <p class="contact-form-text">
+                    I'm always interested in hearing about new projects,
+                    so if you'd like to chat please get in touch.
+                </p>
+                <form class="text-form">
+                    <input type="text" placeholder="Full Name">
+                    <input type="email" placeholder="Email Address">
+                    <textarea cols="30" rows="10">Enter text here</textarea>
+                    <button type="submit" class="contact-form-button">Get in touch</button>
```
Comment (mwanawabangona) [inline index.html:263]: - [ ] good job on the form so far but please try to make it as similar as possible to the figma template. see screenshots below

your form
Screenshot from 2022-07-01 05-44-25

figma template form
Screenshot from 2022-07-01 05-45-04

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
## Status: Changes Required

 ## Hello there @Kwaleyela-Ikafa  👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional briefcase
* ✅  Solid pull request scroll
* ✅  Good use of flexbox 🧐
* ✅  Good quality commits

## Required Changes
 _**Check the comments under the review.**_

- [ ] `required`
- [ ] please add a descriptive readme follow this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -23,13 +23,254 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
...
+                                <li>html</li>
+                            </ul>
+                            <ul class="tag3">
+                                <li>bootstrap</li>
+                            </ul>
+                            <ul class="tag4">
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button type="submit" class="button">See Project</button>
```
Comment (wuletawwonte) [inline index.html:65]: - [ ] [OPTIONAL] Please change the button interaction styling of the `See Project` button to look like what's in the Figma design. 

- [ ] [OPTIONAL] There is no bottom margin for the `See Project` button in the design. So please make sure it has got no bottom margin spacing. 

Buttons

**Review by wuletawwonte — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Kwaleyela-Ikafa 👋🏿👋🏿👋🏿, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**